GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -44,35 +53,39 @@ public static void SetLoadingScreenImagePath()
         /// downloaded from the CDN. Takes in a loadingScreenImagePath, a path to the image shown in the loading scene,
         /// and an assetbundle URL. Replaces the current loading scene with a new one if it exists.
         /// </summary>
-        public static void GenerateLoadingScreenScene(string assetBundleUrl)
+        public static void GenerateLoadingScreenScene()
         {
-            if (!File.Exists(loadingScreenImagePath))
+            if (!File.Exists(LoadingScreenImagePath))
             {
-                Debug.LogErrorFormat("Loading screen image file cannot be found: {0}", loadingScreenImagePath);
+                Debug.LogErrorFormat("Loading screen image file cannot be found: {0}", LoadingScreenImagePath);
             }
             else
             {
                 // Removes the loading scene if it is present, otherwise does nothing.
                 EditorSceneManager.CloseScene(SceneManager.GetSceneByName(LoadingSceneName), true);
 
+                Directory.CreateDirectory(LoadingScreenScenePath);
```

Comment:
I think this line is redundant (can be deleted) according to documentation: "Creates all directories and subdirectories in the specified path unless they already exist."